docs/17-porting-tips.md

```diff
@@ -0,0 +1,333 @@
+    +                    `awful.button`({ }, 5, function(t) `awful.tag.viewprev`(t.screen) end)
+    +                )
+    +
+    +local tasklist_buttons = awful.util.table.join(
+                          awful.button({ }, 1, function (c)
+                                                   if c == `client.focus` then
+                                                       c.minimized = true
+    -mytasklist.buttons = awful.util.table.join(
+                                                       -- Without this, the following
+                                                       -- :isvisible() makes no sense
+                                                       c.minimized = false
+    -                                                  if not c:isvisible() then
+    -                                                      `awful.tag.viewonly`(c:tags()[1])
+    +                                                  if not c:isvisible() and c.`first_tag` then
+    +                                                      c.first_tag:view_only()
+                                                       end
+                                                       -- This will also un-minimize
+                                                       -- the client, if needed
+    -mytasklist.buttons = awful.util.table.join(
+                                                       c:raise()
+                                                   end
+                                               end),
+    -                     `awful.button`({ }, 3, function ()
+    -                                              if instance then
+    -                                                  instance:hide()
+    -                                                  instance = nil
+    -                                              else
+    -                                                  instance = awful.menu.clients({
+    -                                                      theme = { width = 250 }
+    -                                                  })
+    -                                              end
+    -                                          end),
+    +                     `awful.button`({ }, 3, client_menu_toggle_fn()),
+                          `awful.button`({ }, 4, function ()
+                                                   `awful.client.focus.byidx`(1)
+    -                                              if `client.focus` then `client.focus:raise`() end
+                                               end),
+                          awful.button({ }, 5, function ()
+                                                   `awful.client.focus.byidx`(-1)
+    -                                              if `client.focus` then `client.focus:raise`() end
+                                               end))
+
+
+This section is **very important**. This is the section that handle when screens
```

Comment:
handle*s*

However, I would instead propose to replace the sentence: This is where adding and removing screens is handled.